lib/typhoeus/request.rb

```diff
@@ -176,6 +176,14 @@ def encoded_body
       Ethon::Easy::Form.new(nil, options[:body]).to_s
     end
 
+    def to_s
+      <<STR
+#{self.class}
+  url: #{url}
+  options: #{options}
+STR
+    end
+
     private
```

Comment:
I completely agree. Not sure what I was thinking at the time. I'll fix up the PR